buildspec-unit.yml

```diff
@@ -0,0 +1,8 @@
+version: 0.2
+
+phases:
+  build:
+    commands:
+    - pip install --upgrade pip --quiet
+    - pip install tox --quiet
+    - tox -e ${TOX_ENVLIST} -- test/unit
```

Comment:
this project doesn't have a tox config file

Reply:
The tox file is located here https://github.com/aws/sagemaker-tensorflow-container/blob/script-mode/tox.ini